dss/util/checkout.py

```diff
@@ -0,0 +1,125 @@
+import io
+import uuid
+from enum import Enum, auto
+from logging import getLogger
+
+import chainedawslambda
+from chainedawslambda import aws
+from chainedawslambda.s3copyclient import S3ParallelCopySupervisorTask
+
+from cloud_blobstore import BlobNotFoundError, BlobStoreUnknownError
+from cloud_blobstore.s3 import S3BlobStore
+from dss import chained_lambda_clients, DSSException, Config
+from dss.util.aws import get_s3_chunk_size
+from dss.util.bundles import get_bundle, get_bundle_from_bucket
+
+for client_name, client_class in chained_lambda_clients():
```

Comment:
I don't think this is the right place to do this.  It should happen at an entry point for the application.

Reply:
Moving to the daemon code `daemons/dss-checkout-sfn/app.py`